ansys/dpf/core/elements.py

```diff
@@ -251,18 +281,17 @@ def element_by_index(self, index) -> Element:
         return self.__get_element(elementindex=index)  
         
     def add_elements(self, num):     
-        """Add num new elements in the mesh. 
-        add_elements yields num element that the user can fill
-        
+        """Add one ore more elements in the mesh. 
+                
         Parameters
         ----------
         num : int
-            number of elements to add
+            Number of elements to add in the mesh.
         
         Returns
         -------
-        yield element : ElementAdder
```

Comment:
@PipKat I think that we need to mention the fact that it yields, if you look at the example you'll see that the usage is really different than just a getter